test/e2e/sched/install/install_test.go

```diff
@@ -71,9 +74,18 @@ var _ = Describe("[Scheduler] install", func() {
 			err = e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), nroSchedObj)
 			Expect(err).NotTo(HaveOccurred())
 
+			By("checking the NumaResourcesScheduler Deployment is correctly deployed")
 			deploy, err := schedutils.GetDeploymentByOwnerReference(nroSchedObj.UID)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(deploy.Status.ReadyReplicas).To(BeIdenticalTo(int32(1)))
+
+			By("checking the NumaResourcesScheduler CRD is deployed")
+			crd := &apiextensionv1.CustomResourceDefinition{}
+			key := client.ObjectKey{
+				Name: crdName,
+			}
+			err = e2eclient.Client.Get(context.TODO(), key, crd)
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
I'm inclined to think we may need a new utility function. My own threshold is "used in two sites", and we already hit the threshold. More will likely come in the next batch of e2e tests.

Reply:
ok